examples/docker/README.md

```diff
@@ -45,3 +45,9 @@ To downgrade:
 ## Read logs
 
 Logs are available under the `app` service: `docker-compose logs -tf app`.
+
+## Setting up SSL Connection
+
+Create the `server.key` `server.crt` and `ca.crt` files in the `examples/docker/certs` folder.
```

Comment:
The SSL certs will need to be supplied to the container at runtime (probably dumped to the container filesystem); not baked into the image. The docker image will be built _once_ and deployed to various environments; each environment will have different client SSL certs for Postgres.